test/ejabberd_tests/tests/muc_light_http_api_SUITE.erl

```diff
@@ -0,0 +1,181 @@
+        member_is_affiliated(escalus:wait_for_stanza(Alice), Bob),
+        %% XMPP: Alice does NOT recieve an IQ result stanza following
+        %% her HTTP request to invite Bob in story point (*).
+        escalus_assert:has_no_stanzas(Alice)
+      end).
+
+send_message_to_room(Config) ->
+    Domain = <<"localhost">>,
+    Name = <<"wonderland">>,
+    Path = <<"/muc-lights",$/,Domain/binary,$/,
+             Name/binary,$/,"messages">>,
+    Text = <<"Hello everyone!">>,
+    escalus:fresh_story(Config,
+      [{alice, 1}, {bob, 1}, {kate, 1}],
+      fun(Alice, Bob, Kate) ->
+        %% XMPP: Alice creates a room.
+        escalus:send(Alice, muc_light_SUITE:stanza_create_room(undefined,
+            [{<<"roomname">>, Name}], [{Bob, member}, {Kate, member}])),
+        %% XMPP: Get Bob and Kate recieve their affiliation information.
+        [ escalus:wait_for_stanza(U) || U <- [Bob, Kate] ],
+        %% HTTP: Alice sends a message to the MUC room.
+        Body = #{ sender => escalus_client:short_jid(Alice),
+                  message => Text
+                },
+        {{<<"200">>, _}, <<"">>} = rest_helper:post(Path, Body),
+        %% XMPP: Both Bob and Kate see the message.
+        [ see_message_from_user(U, Alice, Text) || U <- [Bob, Kate] ]
+    end).
+
+
+%%--------------------------------------------------------------------
+%% Ancillary (borrowed and adapted from the MUC and MUC Light suites)
+%%--------------------------------------------------------------------
+
+get_disco_rooms(User) ->
+    DiscoStanza = escalus_stanza:to(escalus_stanza:iq_get(?NS_DISCO_ITEMS, []), muc_light_domain()),
+    escalus:send(User, DiscoStanza),
+    Stanza =  escalus:wait_for_stanza(User),
+    XNamespaces = exml_query:paths(Stanza, [{element, <<"query">>}, {attr, <<"xmlns">>}]),
+    true = lists:member(?NS_DISCO_ITEMS, XNamespaces),
+    escalus:assert(is_stanza_from, [muc_light_domain()], Stanza),
+    exml_query:paths(Stanza, [{element, <<"query">>}, {element, <<"item">>}]).
+
+is_room(JID, Item) ->
+    JID == exml_query:attr(Item, <<"jid">>).
+
+see_message_from_user(User, Sender, Contents) ->
+    Stanza = escalus:wait_for_stanza(User),
+    #xmlel{ name = <<"message">> } = Stanza,
+    SenderJID = escalus_utils:jid_to_lower(escalus_utils:get_short_jid(Sender)),
+    From = exml_query:path(Stanza, [{attr, <<"from">>}]),
+    {_, _} = binary:match(From, SenderJID),
+    Contents = exml_query:path(Stanza, [{element, <<"body">>}, cdata]).
+
+member_is_affiliated(Stanza, User) ->
+    MemberJID = escalus_utils:jid_to_lower(escalus_utils:get_short_jid(User)),
+    Data = exml_query:path(Stanza, [{element, <<"x">>}, {element, <<"user">>}, cdata]),
+    MemberJID == Data.
+
+%%--------------------------------------------------------------------
+%% Constants
+%%--------------------------------------------------------------------
+
+muc_light_domain() ->
+    <<"muclight.localhost">>.
```

Comment:
I think we can avoid hardcoding the `localhost` part. Correct domain you can read with:
```erlang
ct:get_config({hosts, mim, domain})
```